proposals/collection-expression-arguments.md

```diff
@@ -303,7 +303,7 @@ l = [with(default)];           // error: ambiguous constructor
 ```
 
 If the target type is a type with a *create method*, then:
-* The *argument list* is the *collection expression* containing the elements only (no arguments), followed by the *argument list*.
+* The *argument list* is the `with()` *argument list* followed by a *collection expression* containing the elements only (no arguments).
```

Comment:
With this approach, it seems like it would only be a small future step to loosen the restrictions on Create methods and allow `IEnumerable<T>` as well as `ReadOnlySpan<T>`.